commit 778c0706f356118c7a5dd11c06725fc6c6b84621
Author: Justus K <justus.k@protonmail.com>
Date:   Tue Sep 22 15:27:53 2020 +0200

    Replace `_mm_movemask_pi8` with the fallback_impl
    
    This has to be done, because `_mm_movemask_pi8` got removed
    from stdarch in https://github.com/rust-lang/stdarch/pull/890

diff --git a/src/codegen/reductions/mask/x86.rs b/src/codegen/reductions/mask/x86.rs
index 2ae4ed8..bcfb1a6 100644
--- a/src/codegen/reductions/mask/x86.rs
+++ b/src/codegen/reductions/mask/x86.rs
@@ -19,13 +19,7 @@ mod avx2;
 /// x86 64-bit m8x8 implementation
 macro_rules! x86_m8x8_impl {
     ($id:ident) => {
-        cfg_if! {
-            if #[cfg(all(target_arch = "x86_64", target_feature = "sse"))] {
-                x86_m8x8_sse_impl!($id);
-            } else {
-                fallback_impl!($id);
-            }
-        }
+        fallback_impl!($id);
     };
 }
 
diff --git a/src/lib.rs b/src/lib.rs
index c3da0d3..cbd438c 100644
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -209,7 +209,6 @@
     link_llvm_intrinsics,
     core_intrinsics,
     stmt_expr_attributes,
-    mmx_target_feature,
     crate_visibility_modifier,
     custom_inner_attributes
 )]
